.github/workflows/main.yml

```diff
@@ -25,6 +25,13 @@ jobs:
       #   run: |
       #     cd quolance-ui
       #     npm test
+      - name: Set up environment variables
+        env:
+          MAILPIT_SMTP_PORT: 1025
```

Comment:
why did we add this to the gh workflow? what happens when we deploy? that would expose the real values no?